src/orange/orangeinp/detail/ProtoBuilder.cc

```diff
@@ -33,7 +33,7 @@ ProtoBuilder::ProtoBuilder(OrangeInput* inp,
     CELER_EXPECT(inp_);
     CELER_EXPECT(opts.tol);
 
-    inp_->tol = opts.tol;
+    inp_->tol = std::move(opts).tol;
```

Comment:
Maybe we should just make the options `const&` since there's no heavyweight data in it anyway.